targets/ramips-mt7621

```diff
@@ -8,7 +8,8 @@ device d-link-dir-860l-b1 dir-860l-b1
 
 ## ZBT
 
-device zbt-wg3526 zbt-wg3526
+device zbt-wg3526-16M zbt-wg3526-16M
+device zbt-wg3526-32M zbt-wg3526-32M
```

Comment:
have you tested that this way to write it generates both images? i would have thought each one needs a completely separate entry in the target file

Reply:
there is no factory - there is only a sysupgrade on build (https://github.com/FreifunkBremen/gluon-site-ffhb/commit/c8dec17d7c75e7649c6753f3405abeef8a37f5a2):
```
ls gluon/lede/bin/targets/ramips/mt762:
lede-ramips-mt7621-zbt-wg3526-32M-squashfs-sysupgrade.bin
```

Reply:
The ZBT-Routers do not have factory-images. They are shipped with OpenWrt/Lede and use only the sysupgrade-file. The problem (the one rotanid mentioned) is that I haven't specified there is no factory firmware for the 16MB-version as like the 32M-version, too. Thus you got an error, but the images are perfectly fine.